docs/OmniMIDI/Soundfonts/recommendations.md

```diff
@@ -6,4 +6,39 @@ Here are some great soundfonts which we recommend you to use for you endevaours
 
     These soundfonts were created by amazing individuals who worked hard to create them and are kind enough to offer them for free for you to use. Please show some love by crediting the creators if you make videos or use them!
 
-> TODO: im not sure what soundfonts would be good. someone pls take over
+## Arachno
+
+Arachno is a really nice soundfont, it's one of the most used soundfonts in the Black MIDI Conmunity.
+It has a lot of instruments, such as drums, percussion and more!
+The download page for arachno can be found on the [Arachnosoft website.](https://www.arachnosoft.com/main/download.php?id=soundfont-sf2)
+
+Arachno is mainly a .sfARK file, however the download above is the .sf2 version which can be used in most synth's, including OmniMIDI
```

Comment:
The official download page isn't the best so if you could, may you also add a picture to show where the link is?

Reply:
Ill add an image